components/RetrievalCommands.tsx

```diff
@@ -0,0 +1,45 @@
+import styles from '@components/RetrievalCommands.module.scss';
+import 'react-tabs/style/react-tabs.css';
+
+import * as React from 'react';
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
```

Comment:
I'm probably going to refactor this out, I don't want to introduce another dependency into our repository, but I'll handle this later.